install/upgrade.php

```diff
@@ -203,7 +237,7 @@ protected function upgrade_2_7() {
 	 *
 	 * @return void
 	 */
-	protected function upgrade_3_4() {
+	protected function migrate_locale_fallback_to_language_description() {
```

Comment:
Not sure the method documentation is still accurate.
Also, do you think that creating a separated method for the term meta migration (as you did with this new method) is a good idea?

Reply:
I agree that you should move the strings migration to a separate method too.
But you must keep the `delete_transient()` and its related comment in the min method as it's not fully related to the fallback migration.